test/ParseableInterface/default-prebuilt-module-location.swift

```diff
@@ -0,0 +1,25 @@
+// 1. Create folders for a) our Swift module, b) the module cache, and c) a
+//    fake resource dir with a default prebuilt module cache inside.
+// RUN: %empty-directory(%t/PrebuiltModule.swiftmodule)
+// RUN: %empty-directory(%t/ModuleCache)
+// RUN: %empty-directory(%t/ResourceDir/%target-os/prebuilt-modules/PrebuiltModule.swiftmodule)
```

Comment:
I don't think this is the correct substitution. `%target-os` is the OS component of the triple, but the platform names match the SDK names instead (e.g. `iphoneos` instead of `ios`). It looks like lit.cfg records this as `config.target_sdk_name`, but never puts it in a substitution.